src/fairseq2/recipes/lm/_online_finetune/_online_dpo.py

```diff
@@ -263,14 +273,24 @@ def __call__(self, prompt_batch: PromptBatch) -> tuple[Tensor, int]:
         rejected_logps, average_rejected_logps = _gather_lprobs_avg(
             rejected_output, rejected_target_batch
         )
-        tgt_logit_entropy = compute_token_level_entropy(
+        chosen_tgt_logit_entropy = compute_token_level_entropy(
             chosen_output.logits, chosen_target_batch.target_mask
         )  # [Batch x Rollouts, 1]
+        rejected_tgt_logit_entropy = compute_token_level_entropy(
+            rejected_output.logits, rejected_target_batch.target_mask
+        )  # [Batch x Rollouts, 1]
+
+        # entropy for all N rollouts
+        logit_entropy = self.get_all_rollouts_entropy(rollouts)
```

Comment:
not sure if we want this. previously `logit_entropy` was computed for the chosen sequences. here i'm computing it for all rollouts